kratos/utilities/read_materials_utility.cpp

```diff
@@ -119,7 +119,12 @@ void ReadMaterialsUtility::AssignPropertyBlock(Parameters Data)
     ModelPart& r_model_part = mrModel.GetModelPart(Data["model_part_name"].GetString());
     const IndexType property_id = Data["properties_id"].GetInt();
     const IndexType mesh_id = 0;
-    Properties::Pointer p_prop = r_model_part.pGetProperties(property_id, mesh_id);
+    Properties::Pointer p_prop = nullptr;
+    if (r_model_part.RecursivelyHasProperties(property_id, mesh_id)) {
+        p_prop = r_model_part.pGetProperties(property_id, mesh_id);
```

Comment:
I think it would be good to issue a warning here, since this is not a standard-case
The checks introduced by @armingeiser a while ago also go into that direction

Reply:
I will approve after you add a warning here :)

Reply:
Ok, but be prepared to have many warnings, because the old tests has properties different than 0